source/Conversation.h

```diff
@@ -75,11 +86,15 @@ class Conversation {
 	int Choices(int node) const;
 	bool IsBranch(int node) const;
 	bool IsAction(int node) const;
-	const ConditionSet &Branch(int node) const;
+	const ConditionSet &Conditions(int node) const;
 	const GameAction &GetAction(int node) const;
 	const std::string &Text(int node, int choice = 0) const;
 	const Sprite *Scene(int node) const;
 	int NextNode(int node, int choice = 0) const;
+	bool ShouldSkipText(const std::map<std::string, int64_t> &vars, int node, int choice = 0) const;
+	bool NodeInRange(int node) const {
+		return node >= 0 && static_cast<unsigned>(node) < nodes.size();
+	}
```

Comment:
We typically don't define functions in header files. It seems to me like this would be better as a namespace function in Conversation and ConversationPanel. (It's also only used once, in ConversationPanel, right now, so should probably check where else it can be put to use.)

Reply:
Apparently I was in the middle of following @tehhowch 's suggestion to make repeated bounds checks into private methods, and I got distracted.

I define methods in headers when they are trivial, like this one is. When compiling without LTO, doing otherwise prevents them from being inlined. Endless Sky gets compiled with LTO (in release builds), so this performance concern is irrelevant.

It can't be an anonymous-namespace function because it accesses a field. Shall I, then, simply move its implementation into `ConversationPanel.cpp`?